cmd2/__init__.py

```diff
@@ -23,6 +23,7 @@
 # Get the current value for argparse_custom.DEFAULT_ARGUMENT_PARSER
```

Comment:
Before merging in this would need a CHANGELOG entry for version 1.2.0 or whatever.